tcms/templates/navbar.html

```diff
@@ -14,7 +14,16 @@
             <a class="navbar-brand" href="/" target="_parent">
                 <img src="{% static 'images/kiwi_h20.png' %}" alt="Kiwi TCMS" />
             </a>
+
+            <b class="caret"></b>
+             <a>
+                <span id="tenant">
+                    {% block tenant %}{% endblock %}
```

Comment:
Let's name the block `customized_logo` and remove all the surrounding HTML. The actual content should be up to the individual plugins.

Reply:
Let's also make the block render an undefined variable named `customized_logo_contents`.  Plugins will be able to override this variable using context processors.